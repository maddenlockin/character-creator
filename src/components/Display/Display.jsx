import React from 'react'

export default function Display({
    judgement,
    judgementArr,
    topCount,
    middleCount,
    bottomCount
}) {
    return (
        <>
            <ul>
                <li key={judgement} aria-label='list items'>
                    '{judgement}'
                </li>
                {/* <li> {judgementArr}  </li> */}
                {judgementArr.map((judgement) =>  (
                    <p>{judgement}</p>
                ))}
            </ul>
            <p>
                The top of your character has been changed {topCount} times, the middle {middleCount} times, and the bottom {bottomCount} times.
            </p>
        </>
    )
}
