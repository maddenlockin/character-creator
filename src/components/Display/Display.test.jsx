import { render } from "@testing-library/react";
import Display from "./Display";

it("should render Display component", () => {
    const { container } = render(
        <Display
        judgements={["ewww", "blah", "ugh"]}
        />
    );
    expect(container).toMatchSnapshot();
});
