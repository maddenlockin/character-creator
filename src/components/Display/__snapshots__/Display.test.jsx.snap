// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Display component 1`] = `
<div>
  <ul>
    <li
      aria-label="list items"
    >
      '
      '
    </li>
    <p>
      ewww
    </p>
    <p>
      blah
    </p>
    <p>
      ugh
    </p>
  </ul>
  <p>
    The top of your character has been changed 
     times, the middle 
     times, and the bottom 
     times.
  </p>
</div>
`;
